examples/custom-hit-detection-renderer.js

```diff
@@ -0,0 +1,111 @@
+import Feature from '../src/ol/Feature.js';
+import Map from '../src/ol/Map.js';
+import View from '../src/ol/View.js';
+import {Circle} from '../src/ol/geom.js';
+import {OSM, Vector as VectorSource} from '../src/ol/source.js';
+import {Style} from '../src/ol/style.js';
+import {Tile as TileLayer, Vector as VectorLayer} from '../src/ol/layer.js';
+import {fromLonLat} from '../src/ol/proj.js';
+
+const columbusCircleCoords = fromLonLat([-73.98189, 40.76805]);
+const labelTextStroke = 'rgba(120, 120, 120, 1)';
+const labelText = 'Columbus Circle';
+
+let pointerOverFeature = null;
+
+const renderLabelText = (ctx, x, y, stroke) => {
+  ctx.fillStyle = 'rgba(255,0,0,1)';
+  ctx.strokeStyle = stroke;
+  ctx.lineWidth = 1;
+  ctx.textAlign = 'center';
+  ctx.textBaseline = 'middle';
+
+  ctx.font = `bold 30px verdana`;
+  ctx.moveTo(x, y);
+  ctx.lineTo(x, y);
+  ctx.filter = 'drop-shadow(7px 7px 2px #e81)';
```

Comment:
Seems like Firefox does not draw a drop-shadow when the stoke / fill is outside the canvas, which means there is no hitdetection on the drop-shadow in Firefox. Doesn't matter much for this example though, just something I noticed.

Reply:
Ok, I have tested this only with Chrome. Thanks.